app/views/hyrax/assets/_show_actions.html.erb

```diff
@@ -31,7 +31,7 @@
             <ul class="dropdown-menu">
               <% presenter.valid_child_concerns.each do |concern| %>
                 <li>
-                  <%= link_to "Attach #{concern.human_readable_type}", polymorphic_path([main_app, :new, :hyrax, :parent, concern.model_name.singular], parent_id: presenter.id) %>
+                  <%= link_to "Attach #{concern.human_readable_type}", polymorphic_path([main_app, :new, :hyrax, :parent, concern.model_name.singular.to_sym], parent_id: presenter.id) %>
```

Comment:
Should we add a conditional here (and in the other places w this change)? Will calling #to_sym break things in production code?

Reply:
From the specs, I observed that this doesn't break anything in the current Rails but I can wrap it up just in case